logic/scripts/update_token_insight.py

```diff
@@ -366,16 +396,186 @@ def fetch_wallet_balance(wallet, label, eth_threshold):
             print (subject)
             sgw.notify_founders(body, subject)
 
-        db.session.add(contact)
-        db.session.commit()
     except Exception as e:
         print e
 
+# Fetches and stores OGN & ETH prices froom CoinGecko
+def fetch_token_prices():
+    print("Fetching token prices...")
+    
+    try:
+        url = "https://api.coingecko.com/api/v3/simple/price?ids=origin-protocol%2Cethereum&vs_currencies=usd"
+        raw_json = requests.get(url)
+        response = raw_json.json()
+
+        if "error" in response:
+            print("Error while fetching balance")
+            print(response["error"]["message"])
+            raise ValueError(response["error"]["message"])
+
+        ogn_price = response["origin-protocol"]["usd"]
+        eth_price = response["ethereum"]["usd"]
+
+        redis_client.set("ogn_price", ogn_price)
```

Comment:
nit: maybe consider using the currency pair as a key in case in the future we decide to store the price of other pairs. For example: "ogn_usd_price" and "eth_usd_price"

Reply:
Will do that.